app-web-frontend/src/reusables/Mood/GetMood.js

```diff
@@ -1,13 +1,18 @@
-import { getUserID, getDoctorID } from "../utils/UserID"
 import { SERVER_ADDRESS } from "../../setupInfo";
 import { getUserAuthToken } from "../utils/AuthToken"
 
 async function getMoodDataFromServer(authToken, startDate, endDate) {
     let formatStartDate = dateToIsoWithoutTime(startDate)
     let formatEndDate = dateToIsoWithoutTime(endDate)
-    const url = `${SERVER_ADDRESS}/user-mood/getByUser/${authToken}/timeframe?start=${formatStartDate}&end=${formatEndDate}`;
+    const url = `${SERVER_ADDRESS}/user-mood/getByUser/timeframe?start=${formatStartDate}&end=${formatEndDate}`;
     try {
-        const response = await fetch(url);
+        const response = await fetch(url, {
+            headers: {
+                'Authorization': `Bearer ${authToken}`
+            },
+            mode: 'no-cors'
```

Comment:
Do we need `no-cors` here? I believe cors should already be properly set up on backend

Reply:
To be fixed later